utils/config_manager.py

```
@@ -607,6 +608,10 @@ def validate_voice_id(self, voice_id):
         if not voice_id:
             return True
 
+        # GPT-SoVITS voice_id（gsv: 前缀）始终有效，不依赖 AUDIO_API_KEY
```

Ruff (0.15.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)